src/main/java/com/missafternoon/budjen/transactions/JSONInterpreter.java

```diff
@@ -0,0 +1,33 @@
+package com.missafternoon.budjen.transactions;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.List;
+
+public class JSONInterpreter {
+
+    private ObjectMapper mapper;
+
+    public JSONInterpreter() {
+        mapper = new ObjectMapper();
+    }
+
+    public String convertToString(List<Transaction> transactions) {
+        try {
+            return mapper.writeValueAsString(transactions);
+        } catch (IOException e) {
+        }
+        throw new RuntimeException();
```

Comment:
This doesn't give you any information about what went wrong. Have a closer look at my suggestion here https://github.com/JStonham/budjen/pull/30#discussion_r244646124